core/src/apps/benfen/tx_parser.py

```
@@ -0,0 +1,432 @@
+import binascii
```

Ruff (0.8.2)

The file is executable but no shebang is present

(EXE002)